packages/monitoring/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "monitoring",
+  "version": "1.0.0",
+  "description": "Threefold monitoring package",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "example": "yarn ts-node ./example/index.js",
```

```suggestion
    "example": "yarn ts-node ./example/index.ts",
```

Example should be executed directly from typescript code